_episodes/09-cmorization.md

```diff
@@ -141,38 +147,52 @@ name that was created to store the raw observation data files, i.e.
 If everything is okay, the output should look something like this:
 
 ~~~
-...
-... Starting the CMORization Tool at time: 2022-07-26 14:02:16 UTC
+... Writing program log files to:
+/scratch/b/b309059/esmvaltool_output/data_formatting_20240527_132448/run/main_log.txt
```

Comment:
Can all references to b309059 be changed to `username` so it is less confusing and consistent with other episodes?